content/blog/2018/egress-monitoring-access-control/index.md

```diff
@@ -0,0 +1,613 @@
+[edition.cnn.com/politics](https://edition.cnn.com/politics) automatically.
+
+The organization is resolved to prevent any tampering with the new policy. It decides to put mechanisms in place that
+will prevent any possibility for a malicious application to access the forbidden topic.
+
+## Related tasks and examples
+
+The [Control Egress Traffic](/docs/tasks/traffic-management/egress/) task demonstrates how external (outside the
+  Kubernetes cluster) HTTP and HTTPS services can be accessed by applications inside the mesh. The [Configure an Egress Gateway](/docs/examples/advanced-gateways/egress-gateway/) example describes how to configure Istio to direct egress
+  traffic through a dedicated gateway service called _egress gateway_. The
+  [Egress Gateway with TLS Origination](/docs/examples/advanced-gateways/egress-gateway-tls-origination/) example demonstrates how
+  to allow applications to send HTTP requests to external servers that require HTTPS, while directing traffic through
+  egress gateway.
+
+The [Collecting Metrics and Logs](/docs/tasks/telemetry/metrics-logs/) task describes how to configure metrics and logs
+ for services in a mesh. The [Visualizing Metrics with Grafana](/docs/tasks/telemetry/using-istio-dashboard/) describes
+ the Istio Dashboard to monitor mesh traffic. The [Basic Access Control](/docs/tasks/policy-enforcement/denial-and-list/)
+ task shows how to control access to in-mesh services. The
+ [Secure Access Control](/docs/tasks/security/secure-access-control/) task shows how to configure
+ access policies using black or white list checkers. As opposed to the telemetry and security tasks above, this blog
+ post describes Istio's monitoring and access policies applied exclusively to the egress traffic.
+
+## Before you begin
+
+Follow the steps in the [Egress Gateway with TLS Origination](/docs/examples/advanced-gateways/egress-gateway-tls-origination/) example, **with mutual TLS authentication enabled**, without
+the [Cleanup](/docs/examples/advanced-gateways/egress-gateway-tls-origination//#cleanup) step.
+After you accomplish this, you will be able
+to access [edition.cnn.com/politics](https://edition.cnn.com/politics) from an in-mesh container that has _curl_
+installed. In the instructions of this blog post we assume that the `SOURCE_POD` environment variable contains the
+source pod's name and the container's name is `sleep`.
+
+## Configure monitoring and access policies
+
+Note that since you want to accomplish your tasks in a _secure way_, you must direct egress traffic through
+_egress gateway_, as described in the [Egress Gateway with TLS Origination](/docs/examples/advanced-gateways/egress-gateway-tls-origination/)
+task. The _secure way_ here means that you want to prevent malicious applications from bypassing Istio monitoring and
+policy enforcement.
+
+According to the scenario of this blog post, the organization performed the instructions in the
```

Comment:
To help readers identify with your message, consider including the reader in the narrative. This technique can be effective to keep the text clear and simple.
```suggestion
According to our scenario, the organization performed the instructions in the
```

Reply:
@rcaballeromx According to the istio.io style guide (https://istio.io/about/contribute/style-guide/#avoid-using-we), "we" should be avoided, so I assumed that "our" should be avoided too.